### YamlMime:ManagedReference
items:
- uid: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs
  id: WorkflowApplicationUnhandledExceptionEventArgs
  children:
  - System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource
  - System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId
  - System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException
  langs:
  - csharp
  name: WorkflowApplicationUnhandledExceptionEventArgs
  nameWithType: WorkflowApplicationUnhandledExceptionEventArgs
  fullName: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs
  type: Class
  summary: "Fornece informações sobre uma exceção não tratada ocorrida em uma instância de fluxo de trabalho."
  remarks: "Se uma exceção é gerada por uma atividade e sem tratamento, o comportamento padrão é encerrar a instância de fluxo de trabalho. Se um <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>manipulador estiver presente, ele pode substituir esse comportamento padrão.</xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> Este manipulador o autor do host de fluxo de trabalho uma oportunidade para fornecer o tratamento apropriado, como o registro em log, anulando o fluxo de trabalho, cancelando o fluxo de trabalho ou encerrando o fluxo de trabalho personalizado."
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The WorkflowApplicationUnhandledExceptionEventArgs are inspected to provide information about the exception, and the workflow is terminated.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#1](~/add/codesnippet/csharp/t-system.activities.work_0_1.cs)]"
  syntax:
    content: 'public class WorkflowApplicationUnhandledExceptionEventArgs : System.Activities.WorkflowApplicationEventArgs'
  inheritance:
  - System.EventArgs
  - System.Activities.WorkflowApplicationEventArgs
  implements: []
  inheritedMembers:
  - System.Activities.WorkflowApplicationEventArgs.GetInstanceExtensions``1
  - System.Activities.WorkflowApplicationEventArgs.InstanceId
  platform:
  - net462
- uid: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource
  id: ExceptionSource
  parent: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs
  langs:
  - csharp
  name: ExceptionSource
  nameWithType: WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource
  fullName: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource
  type: Property
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Obtém a atividade que é a origem da exceção sem tratamento."
  remarks: "Se uma exceção é gerada por uma atividade e sem tratamento, o comportamento padrão é encerrar a instância de fluxo de trabalho. Se um <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>manipulador estiver presente, ele pode substituir esse comportamento padrão.</xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> Este manipulador o autor do host de fluxo de trabalho uma oportunidade para fornecer o tratamento apropriado, como o registro em log, anulando o fluxo de trabalho, cancelando o fluxo de trabalho ou encerrando o fluxo de trabalho personalizado."
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> are inspected to provide information about the exception, and the workflow is terminated.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#1](~/add/codesnippet/csharp/eda69030-f6e5-45e1-954d-_1.cs)]"
  syntax:
    content: public System.Activities.Activity ExceptionSource { get; }
    return:
      type: System.Activities.Activity
      description: "Uma atividade."
  overload: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource*
  exceptions: []
  platform:
  - net462
- uid: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId
  id: ExceptionSourceInstanceId
  parent: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs
  langs:
  - csharp
  name: ExceptionSourceInstanceId
  nameWithType: WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId
  fullName: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId
  type: Property
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Obtém o identificador exclusivo da instância de atividade que é a origem da exceção sem tratamento."
  remarks: "Se uma exceção é gerada por uma atividade e sem tratamento, o comportamento padrão é encerrar a instância de fluxo de trabalho. Se um <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>manipulador estiver presente, ele pode substituir esse comportamento padrão.</xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> Este manipulador o autor do host de fluxo de trabalho uma oportunidade para fornecer o tratamento apropriado, como o registro em log, anulando o fluxo de trabalho, cancelando o fluxo de trabalho ou encerrando o fluxo de trabalho personalizado."
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> are inspected to provide information about the exception, and the workflow is terminated.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#1](~/add/codesnippet/csharp/28b0f72c-0ddb-46ea-94fd-_1.cs)]"
  syntax:
    content: public string ExceptionSourceInstanceId { get; }
    return:
      type: System.String
      description: "Um identificador de instância de atividade que é a origem da exceção sem tratamento."
  overload: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId*
  exceptions: []
  platform:
  - net462
- uid: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException
  id: UnhandledException
  parent: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs
  langs:
  - csharp
  name: UnhandledException
  nameWithType: WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException
  fullName: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException
  type: Property
  assemblies:
  - System.Activities
  namespace: System.Activities
  summary: "Obtém o <xref:System.Exception>que não foi tratada pela instância de fluxo de trabalho.</xref:System.Exception>"
  remarks: "Se uma exceção é gerada por uma atividade e sem tratamento, o comportamento padrão é encerrar a instância de fluxo de trabalho. Se um <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A>manipulador estiver presente, ele pode substituir esse comportamento padrão.</xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> Este manipulador o autor do host de fluxo de trabalho uma oportunidade para fornecer o tratamento apropriado, como o registro em log, anulando o fluxo de trabalho, cancelando o fluxo de trabalho ou encerrando o fluxo de trabalho personalizado."
  example:
  - "The following example invokes a workflow that throws an exception. The exception is unhandled by the workflow and the <xref:System.Activities.WorkflowApplication.OnUnhandledException%2A> handler is invoked. The <xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs> are inspected to provide information about the exception, and the workflow is terminated.  \n  \n [!code-cs[CFX_WorkflowApplicationExample#1](~/add/codesnippet/csharp/6087d33d-eea3-44f0-8b12-_1.cs)]"
  syntax:
    content: public Exception UnhandledException { get; }
    return:
      type: System.Exception
      description: "O <xref:System.Exception>que não foi tratada pela instância de fluxo de trabalho.</xref:System.Exception>"
  overload: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Activities.WorkflowApplicationEventArgs
  isExternal: false
  name: System.Activities.WorkflowApplicationEventArgs
- uid: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource
  parent: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs
  isExternal: false
  name: ExceptionSource
  nameWithType: WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource
  fullName: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource
- uid: System.Activities.Activity
  parent: System.Activities
  isExternal: false
  name: Activity
  nameWithType: Activity
  fullName: System.Activities.Activity
- uid: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId
  parent: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs
  isExternal: false
  name: ExceptionSourceInstanceId
  nameWithType: WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId
  fullName: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException
  parent: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs
  isExternal: false
  name: UnhandledException
  nameWithType: WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException
  fullName: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource*
  parent: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs
  isExternal: false
  name: ExceptionSource
  nameWithType: WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource
- uid: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId*
  parent: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs
  isExternal: false
  name: ExceptionSourceInstanceId
  nameWithType: WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId
- uid: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException*
  parent: System.Activities.WorkflowApplicationUnhandledExceptionEventArgs
  isExternal: false
  name: UnhandledException
  nameWithType: WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException
